packages/xo-web/src/xo-app/proxies/deploy-proxy.js

```diff
@@ -228,7 +228,17 @@ const Modal = decorate([
   ),
 ])
 
-const deployProxy = ({ license, proxy }) => {
+const deployProxy = async ({ proxy }) => {
+  const licenses = await getLicenses({ productType: 'xoproxy' })
+  const license =
+    licenses &&
+    licenses.find(
+      license =>
+        !(license.expires < Date.now()) &&
+        (license.boundObjectId === get(() => proxy.vmUuid) ||
+          license.boundObjectId === undefined)
```

Comment:
Why the second part of the test?

Reply:
In case of redeploying a proxy with a proxy VM with no license.